### ISSUE TITLE
Cannot add extensions to Server or Database dashboards on March-insider build

### ISSUE THREAD

**Issue by yualan** [Microsoft/sqlopsstudio#996]
- SQL Operations Studio Version: 0.27.2-insider w/ Windows Setup

- SQL Server Instance: SQL Server 2017 Developer Edition on Windows (localhost)

Steps to Reproduce:
 - Installed all 3 extensions on Extensions Marketplace
 - Reset SQL Operations Studio
 - Connect to localhost server 
 - Right click server and click Manage
 - Click + Tab, no extensions available

![image](https://user-images.githubusercontent.com/30873802/37936492-a6d48a70-310a-11e8-84d9-9b2b703d79c4.png)

![image](https://user-images.githubusercontent.com/30873802/37936540-d16333cc-310a-11e8-8c5d-79950cf884b3.png)

**Comment by kburtram** [comment]
@yualan the extensions appear to be in a state that requires SQL Ops to restart.  If you click the "Reload" button in the Extension Manager (per the attached screenshot) SQL Ops should restart.  After that, does the issue still repro?

**Comment by yualan** [comment]
@kburtram  Yes, I clicked reload for each extension and the add extension does not appear. Will try testing on other machines and servers.

**Comment by anthonydresser** [comment]
I've seen the problem you described @yualan regarding extensions not reloading/activating. I wasn't able to reproduce since i had it happen. Just wanted to throw this in that I experienced this a couple weeks ago, so it's not a new thing. But it is intermittent.

**Comment by kburtram** [comment]
@yualan hmm, I haven't seen this behavior yet.  Could you try clearing your installed extensions and see if that helps?  You can do that by removing this directory `C:\Users\[USERNAME]\.sqlops\extensions`.

**Comment by kburtram** [comment]
This issue has been confirmed by customer and I have a repro.  It looks like there is an issue with the VS Code engine version conflicting with SQL Ops engine versions cause failure to load extensions.  SQL Ops version is 0.27 and there is a check against VS Code version >1.19.  Reverting https://github.com/Microsoft/sqlopsstudio/pull/990 seems to workaround the issue by not enforcing this check.  I'm queuing a new build with #990 reverted from release/0.27.

**Comment by kburtram** [comment]
0.27.3 release is updated with rollback of #990.  I can't repro this issue on the current release build.  I'll move this to April milestone to reapply #990 with necessary fixes.
